addons/mrp/static/src/mrp_forecasted/forecasted_details.js

```diff
@@ -0,0 +1,12 @@
+/** @odoo-module **/
+
+import { ForecastedDetails } from '@stock/stock_forecasted/forecasted_details';
+import { StockForecasted } from '@stock/stock_forecasted/stock_forecasted';
+
+export default class MRPForecastedDetails extends ForecastedDetails {
+
+    displayReserve(line) {
+        return line.move_out && (line.move_out.picking_id || line.move_out.raw_material_production_id) && !line.in_transit;
+    }
+}
+StockForecasted.components.ForecastedDetails = MRPForecastedDetails;
```

Comment:
There is a reason of doing that instead of a patch ? 
This override will be lost (or will erase other overrides, depending of the way it's done) if we have to override `ForecastedDetails` in a module who doesn't require MRP